src/main/scala/cromwell/binding/Workflow.scala

```diff
@@ -93,7 +93,8 @@ object Workflow {
     }
     val workflowOutputsDecls = ast.findAsts(AstNodeName.WorkflowOutput) map { wfOutput =>
       val wildcard = Option(wfOutput.getAttribute("wildcard")).map(_.sourceString).getOrElse("").nonEmpty
-      WorkflowOutputDeclaration(wfOutput.getAttribute("fqn").sourceString, wildcard)
+      val outputFqn = s"$name.${wfOutput.getAttribute("fqn").sourceString}"
```

Comment:
IMHO this is not a great use case for interpolation, almost nothing in the template and nontrivial expressions being interpolated